### ISSUE TITLE
Support custom avatars

### ISSUE THREAD

**Issue by mihok** [minimalchat/daemon#9]
Today we don't care about Operator avatars.

Avatars add a level of personality to the chat window from the user/client perspective. This could mean we either create a specific endpoint to handle image upload, which would require us to save the image data somewhere (database, file store, etc?) Or save just a URI to an image (maybe also support `file:///`?)

**Comment by broneks** [comment]
we could also just use gravatar

**Comment by mihok** [comment]
Good idea!! This could be the solution for now, gravatar, we'll have to collect an email in the config.json or something
